config-overrides.js

```diff
@@ -8,6 +8,12 @@ module.exports = {
     // ...add your webpack config
     // suppress grpc warnings
     config.module.exprContextCritical = false
+
+    // remove hot-loader
+    if (process.env.REACT_APP_HOT_LOADING === 'false') {
```

Comment:
use typedEnv if possible

Reply:
cannot use typed env for webpack config file